docs/resources/hotels.md

```diff
@@ -1,5 +1,14 @@
 # Hotels
 
+| APIs                                                                                                                                                          | Description                                                                             |
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| [Amadeus Hotel APIs](https://developers.amadeus.com/self-service/category/hotel) | Lets you search, compare and book rooms at over 350 of the world’s top hotel chains and enrich your product with detailed information, descriptions, and ratings.                |
```

Comment:
Do we want to keep this in the table or its a normal sentence out of the table?

Reply:
Thanks @tsolakoua , formatting error - removed.